dist/docker/Dockerfile

```diff
@@ -46,6 +47,16 @@ RUN apt-get update -y && \
   # Clean up the lists work
   rm -rf /var/lib/apt/lists/*
 
+# Install .NET Core SDK
+RUN wget -q https://packages.microsoft.com/config/ubuntu/19.04/packages-microsoft-prod.deb \
+      -O packages-microsoft-prod.deb && \
+    dpkg -i packages-microsoft-prod.deb && \
+    rm packages-microsoft-prod.deb && \
+    apt-get update -y && \
+    apt-get install -y \
+      apt-transport-https \
+      dotnet-sdk-3.1
```

Comment:
Using the recently released 3.1 makes sense to me as that's the LTS version. However, we should probably move to using it everywhere else first, right? https://github.com/pulumi/scripts/blob/ae6f56d5f37fb5ac0bc232d746a768b137f21943/ci/install-common-toolchain.sh#L87-L99

Reply:
I don't have any objection, but I'd love for @pgavlin to weigh in. Since one is the "runtime dependency for Pulumi apps users are running" and the other is a "developer-machine dependency" we are installing when building Pulumi locally.

Having the versions be different is probably asking for trouble. But it might also lead to unintended consequences. But upgrading the .NET SDK we use to the LTS version seems like a general thing we should be doing. Right?

Reply:
I'll drop this down to 3.0 to match everything else for the time being, thanks Justin.